app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = []
+    for person in people:
+        result.append(Person(person["name"], person["age"]))
+
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife = Person.people[person["wife"]]
+        elif person.get("husband"):
+            Person.people[person["name"]].husband = \
+                Person.people[person["husband"]]
```

Comment:
try not to use "\" in ur code, u can make a variable for Person.people[person["name"]] or break by [] brackets